regimes/mx/invoice_validator.go

```diff
@@ -77,9 +81,37 @@ func (v *invoiceValidator) validLine(value interface{}) error {
 			validation.Required,
 			validation.Skip, // Prevents each tax's `ValidateWithContext` function from being called again.
 		),
+		validation.Field(&line.Item, validation.By(v.validItem)),
 	)
 }
 
+func (v *invoiceValidator) validItem(value interface{}) error {
```

Comment:
Rather than just validating inside the context of an invoice, perhaps they could be validated in general. I can imagine a scenario whereby a user has a database of `org.Item` to be used inside invoices. It makes sense that they should be validatable in the context of a tax regime as independent objects. (Let me know if you'd like to me to and suggest the change in a PR into this one)

Reply:
Sure! I suppose this comment was your motivation for the changes you added in #168. Happy to refactor to validate directly rather than in the context of the invoice as soon as the changes are merged!

Reply:
Cool! That's now merged!

Reply:
Addressed in 7a6ab005bc1640570fa51ac6a1868ad7ff2fb077